connection.js

```diff
@@ -218,6 +222,34 @@ class Connection {
             }
         }
     };
+    dumpStatus() {
+        conn_log('Dumping status of all connected games');
+        for (let game_id in this.connected_games) {
+            let game = this.connected_games[game_id];
+            let msg = [];
+            msg.push('game_id=' + game_id + ':');
+            if (game.state == null) {
```

Comment:
Since we're doing a general style overhaul to use === and !== everywhere would you change this over to ===? Unless there's a reason we shouldn't do the overhaul?

Reply:
Done